src/routines/routines.ycp

```diff
@@ -912,12 +912,12 @@ define list<string> GetAllInterfaces()
 
 boolean SetLinkUp( string dev_name)
 {
-    return Run( sformat("ip link set %1 up", dev_name) );
+    return Run( sformat("ifup %1", dev_name) );
 }
 
 boolean SetLinkDown( string dev_name)
 {
-    return Run( sformat("ip link set %1 down", dev_name) );
+    return Run( sformat("ifdown %1", dev_name) );
```

Comment:
bcs setting link up / down is just about setting a flag. But we need to reset the device's configuration completely. Otherwise it could happen that when switching from DHCP -> static, the DHCP address could stay on the device.

This part of the patch will be ported into SLE-12.